lib/resin-lint.ts

```diff
@@ -46,65 +65,74 @@ const prettierConfigPath = path.join(__dirname, '../config/.prettierrc');
  * package.json is). This function takes a path and propagates upwards
  * until it contains a package.json
  */
-const getPackageJsonDir = function(dir: string): string {
-	const name = findFile('package.json', dir);
+const getPackageJsonDir = async (dir: string): Promise<string> => {
+	const name = await findFile('package.json', dir);
 	if (name === null) {
 		throw new Error('Could not find package.json!');
 	}
 	return path.dirname(name);
 };
 
-const read = function(filepath: string): string {
-	const realPath = fs.realpathSync(filepath);
-	return fs.readFileSync(realPath).toString();
+const read = async (filepath: string): Promise<string> => {
+	const realPath = await realpathAsync(filepath);
```

Comment:
I wonder why calling `realpath` is needed/desired, and therefore why this `read` function is needed... But this PR doesn't need to change it.

Reply:
Yeah I'm also not sure and considered removing it but without knowing why it was added I'm a bit unsure about it. Maybe bundling it with a major version that also drops node 8 support makes sense